### ISSUE TITLE
Hang while 'Checking your identity' on Linux

### ISSUE THREAD

**Issue by niobium93** [timiimit/UT4UU-Public#15]
**Describe the bug**
When trying to log in, the game hangs on the `Checking your identity` screen.

**To Reproduce**
1. Run the game
2. Paste authorization code
3. Click `Add Account`
4. Click `Use`
5. The game hangs on the `Checking your identity` screen

**Expected behavior**
Game should proceed to log me in.

**Screenshots or videos**
![Screenshot_20230109_201531](https://user-images.githubusercontent.com/1523948/211378791-656dbaa5-a22d-465e-84ca-bddc3bbdd40b.png)

**System Information (please complete the following information):**
 - OS: Arch Linux
 - Build Configuration: Game
 - Version: v10.1.1

**Additional context**
Game repeatedly spams messages like this to the terminal:
```
[2023.01.09-18.15.41:545][734]LogXmpp:Warning: Certificate verification failure: self signed certificate in certificate chain
[2023.01.09-18.15.41:545][734]LogXmpp:Warning: Issuer: /C=US/O=DigiCert Inc/OU=www.digicert.com/CN=DigiCert Global Root G2
[2023.01.09-18.15.41:545][734]LogXmpp:Warning: Subject: /C=US/O=DigiCert Inc/OU=www.digicert.com/CN=DigiCert Global Root G2
[2023.01.09-18.15.41:545][734]LogXmpp:Warning: Fingerprint SHA256: cb3ccbb76031e5e0138f8dd39a23f9de47ffc35e43c1144cea27d46a5ab1cb5f
[2023.01.09-18.15.41:545][734]LogXmpp:Warning: Fingerprint SHA1: df3c24f9bfd666761b268073fe06d1cc8d4f82a4

```

I was asked to check if I have certificates installed in `/etc/ssl` and, yes, I seem to. SSL in general works fine in other software.

**Comment by timiimit** [comment]
Thank you for reporting. I think i tracked down the issue. I released [v10.1.2](https://github.com/timiimit/UT4UU-Public/releases/tag/v10.1.2) which should fix this.
Please let me know if thats the case.

**Comment by niobium93** [comment]
That fixed it! I can log in fine now. Thanks for your work!
